multicluster/controllers/multicluster/node_controller.go

```diff
@@ -142,7 +143,17 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 			return ctrl.Result{}, err
 		}
 	}
-	return ctrl.Result{}, nil
+
+	// When a Node is added to a member cluster or removed from a member cluster.
```

Comment:
I feel this comment should be updated,  this block also handles Node update case.
Can we compare existing CIDRs so we can skip refresh Gateway unnecessarily?

Reply:
The reason I used node list rather than other annotations such as "lastUpdateTime" is that we will compare the node list to skip the unnecessary refresh in the `updateActiveGateway` method. 

If we need to compare the CIDRs, maybe we need to set the store the CIDRs in the Gateway.Spec or Annotations. I'm not sure if it's necessary since we store it in the ClusterInfo.

Reply:
Do you guys think we can just ask users to provide the PodCIDRs? I want not to add much complexity for auto-discovering Pod CIDRs? @luolanzone

Reply:
Yes, I think this is also workable. Maybe add a config item `clusterCIDRs` the same as Antrea agent?

Reply:
@luolanzone @jianjuns Maybe we can add a new field `PodCIDRs` in struct `options` in`multicluster/cmd/multicluster-controller/options.go`. And the user must edit the `antrea-multicluster-member.yml` to specify the CIDR.

The CIDR must be the cluster Pod CIDRs or a superset of it, should we add a check about it?

Reply:
I think no need to check. But wonder if we should check no overlapping among clusters. At least we should check in each cluster that remote cluster CIDRs cannot overlap with the local cluster, and reject that import if it does overlap. We can do a separate PR for that.

Reply:
@hjiajing  Yes, I think a new option field is required for PodCIDRs, and you should also add a config item here https://github.com/antrea-io/antrea/blob/main/multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go#L38.

Reply:
@luolanzone Thanks for the reminder. The item `PodCIDRs` was added to `MulticlusterConfig`.